src/programs/city/mine.js

```diff
@@ -100,7 +101,34 @@ class CityMine extends kernel.process {
       return
```

Comment:
Is this early return still needed with the new one on line 129?

Reply:
Good catch